lib/generators/manageiq/plugin/templates/lib/%plugin_path%/engine.rb

```diff
@@ -11,5 +11,13 @@ def self.vmdb_plugin?
     def self.plugin_name
       _('<%= plugin_human_name %>')
     end
+
+    def self.init_loggers
+      $<%= plugin_name %>_log ||= Vmdb::Loggers.create_logger("<%= plugin_name %>.log")
```

Comment:
Should we default this line to be commented out to start?  Just seems like it makes sense to have this opt in to start, and if a more isolated logger is required by the plugin author, the interface is easily adjusted in the code base.

Just trying to avoid "log-splosion" in the `log/` dir, with a bunch of unused files and debugging turns into checking a bunch of files that in fact are just empty.  But possibly this is a necessary concern.

Reply:
Coincidentally, just talked about this with @agrare and yeah I agree.  I am going to change this to commented out.

Reply:
Yeah I think the ideal would be command line option for the provider generator like:
```
rails generate manageiq:plugin [--logger]/[--no-logger]
```

And then this could become:
```
<% if options[:logger] -%>
    def self.init_loggers
      ...
    end
<% end -%>
```